src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -1046,4 +1047,11 @@ private InstructorFeedbackEditPage getFeedbackEditPageOfCourseWithoutQuestions()
         return loginAdminToPage(feedbackPageLink, InstructorFeedbackEditPage.class);
     }
 
+    private boolean verifyIfVisibilityOptionsDivHasAlertClassEnabled() {
```

Comment:
This looks out of place here; see if you can move it into `InstructorFeedbackEditPage` in a more general manner (e.g. question number can be a parameter). Try to follow the naming conventions there too.